anvil/src/config.rs

```diff
@@ -106,6 +117,160 @@ pub struct NodeConfig {
     pub transaction_order: TransactionOrder,
+    /// Filename to write anvil output as json
+    pub config_out: Option<String>,
+}
+
+impl NodeConfig {
+    fn as_string(&self, fork: Option<&ClientFork>) -> String {
+        let mut config_string: String = "".to_owned();
+        let _ = write!(config_string, "\n{}", Paint::green(BANNER));
+        let _ = write!(config_string, "\n    {}", VERSION_MESSAGE);
+        let _ = write!(
+            config_string,
+            "\n    {}",
+            Paint::green("https://github.com/foundry-rs/foundry")
+        );
+
+        let _ = write!(
+            config_string,
+            r#"
+
+Available Accounts
+==================
+"#
+        );
+        let balance = format_ether(self.genesis_balance);
+        for (idx, wallet) in self.genesis_accounts.iter().enumerate() {
+            let _ = write!(config_string, "\n({}) {:?} ({} ETH)", idx, wallet.address(), balance);
+        }
+
+        let _ = write!(
+            config_string,
+            r#"
+
+Private Keys
+==================
+"#
+        );
+
+        for (idx, wallet) in self.genesis_accounts.iter().enumerate() {
+            let hex = hex::encode(wallet.signer().to_bytes());
+            let _ = write!(config_string, "\n({}) 0x{}", idx, hex);
+        }
+
+        if let Some(ref gen) = self.account_generator {
+            let _ = write!(
+                config_string,
+                r#"
+
+Wallet
+==================
+Mnemonic:          {}
+Derivation path:   {}
+"#,
+                gen.phrase,
+                gen.get_derivation_path()
+            );
+        }
+
+        let _ = write!(
+            config_string,
+            r#"
+
+Base Fee
+==================
+{}
+"#,
+            Paint::green(format!("\n{}", self.base_fee))
+        );
+        let _ = write!(
+            config_string,
+            r#"
+Gas Price
+==================
+{}
+"#,
+            Paint::green(format!("\n{}", self.gas_price))
+        );
+
+        let _ = write!(
+            config_string,
+            r#"
+Gas Limit
+==================
+{}
+"#,
+            Paint::green(format!("\n{}", self.gas_limit))
+        );
+
+        if let Some(fork) = fork {
+            let _ = write!(
+                config_string,
+                r#"
+Fork
+==================
+Endpoint:       {}
+Block number:   {}
+Block hash:     {:?}
+Chain ID:       {}
+
+"#,
+                fork.eth_rpc_url(),
+                fork.block_number(),
+                fork.block_hash(),
+                fork.chain_id()
+            );
+        }
+
+        config_string
+    }
+
+    fn as_json(&self, fork: Option<&ClientFork>) -> String {
+        let config_as_json: Value;
+        let mut wallet_description = HashMap::new();
+        let mut available_accounts = Vec::with_capacity(self.genesis_accounts.len());
+        let mut private_keys = Vec::with_capacity(self.genesis_accounts.len());
+
+        for (_, wallet) in self.genesis_accounts.iter().enumerate() {
```

Comment:
Don't need the iter/enumerate, can just do "for wallet in &self.genesis_accounts"